tiled/catalog/adapter.py

```diff
@@ -1083,6 +1121,212 @@ async def create_node(
             structure_family, metadata, key=key, specs=specs, data_sources=data_sources
         )
 
+    async def get_dataset_items(
+        self,
+        keys=None,
+        return_adapters=False,
+        adapter_keys=None,
+        default_dim0="time",
```

Comment:
This is a bluesky assumption. The `bluesky-tiled-plugins` can provide this; Tiled should not elevate any particular string as a default.